core/dbt/task/debug.py

```diff
@@ -208,7 +210,7 @@ def _choose_profile_names(self) -> Optional[List[str]]:
                     '\n'.join(' - {}'.format(o) for o in profiles)
                 ))
                 return profiles
-        return None
+        return []
```

Comment:
`_load_profile()` expects the return of this method to be a list: https://github.com/fishtown-analytics/dbt/blob/4e58589afdbb5e74701753e239536e318c5d8db7/core/dbt/task/debug.py#L255-L257